### PR TITLE
Storm Surge REU Website Draft

### PR DESCRIPTION
**Pull request by emilycorcoran7**

### CODE REVIEW

File: content/projects/2022-storm-surge/index.md [hunk 1]
```diff
@@ -4,6 +4,123 @@ date: 2022-06-27T11:36:49-04:00
...
+summary: 'Numerical models of coastal hydrodynamics play a vital role in understanding hurricane storm surges, particularly as the climate changes. However, uncertainties in model parameters and their representations, e.g., bottom stress and surface wind stress, limit the confidence of model results. Data is becoming increasingly available but also contains uncertainties. In this project we will explore methods of data assimilation, which utilize information about the uncertainties in both modeled and observed data to improve estimates of the coastal system state.'
```
Comment (lruthotto) [inline content/projects/2022-storm-surge/index.md:7]: change tempus in the end.

File: content/projects/2022-storm-surge/index.md [hunk 2]
```diff
@@ -4,6 +4,123 @@ date: 2022-06-27T11:36:49-04:00
...
 tags: ["Summer 2022"]
 ---
+
+ # Data Assimilation for Hurricane Storm Surge Modeling
+
+This post was written by Emily Corcoran, Hannah Park- Kaufmann, and Logan Knudsen. The project was advised by Dr. Talea Mayo.  Our team has also created a [midterm presentation](https://github.com/hakuupi/emory-reu-ret-website/blob/main/content/projects/2022-storm-surge/img/data_assimilation_for_glacier_modeling.pdf), blitz video, poster, paper, and has written code.
```
Comment (lruthotto) [inline content/projects/2022-storm-surge/index.md:13]: If you have personal websites, please link here.

**Comment by emilycorcoran7** [comment]
Blog Post July 22, 2022

**Comment by hakuupi** [comment]
Storm Surge Project Website!

**Review by lruthotto — APPROVED** [review]
Hi y'all,

Thanks for working on this. I think this is ready to go. I may do some small changes before publishing it.

**Pull request merged**